### ISSUE TITLE
Something wrong with it.

### ISSUE THREAD

**Issue by samuth1** [rA9stuff/LeetDown#119]
My mac: 2021 M1 Pro
mac os: Monterey 12.6
My ipad mini 2.
the error show:
[+]  Waiting for a device in DFU Mode
[+]  iPad Mini 2 (WiFi) with ECID: 8481096735536 is connected in wrong mode, please place it in DFU mode to proceed

**Comment by rA9stuff** [comment]
Click on `need help entering DFU mode?` button and follow the instructions.

**Comment by samuth1** [comment]
Yes, My ipad was already DFU mode. and i have tried the leeDown 2.1. It's not work.:(

**Comment by samuth1** [comment]
Yes my iPad is already DFU mode. And I have tryed the leedown 2.1. Edition.  It doesn’t work.

**Comment by rA9stuff** [comment]
send the output of `irecovery -q`

**Comment by makkussu** [comment]
Have you tried to use USB-A to Lightning connector? It has fixed the issue for me

**Comment by xttxqjfg** [comment]
> Have you tried to use USB-A to Lightning connector? It has fixed the issue for me. For some reason USB-C to Lightning does not put device into DFU mode but recovery mode

USB-A is right！
